src/index.html

```diff
@@ -7,23 +7,27 @@
 		<meta name="viewport" content="width=device-width,initial-scale=1.0" />
 
 		<meta name="author" content="P3K Team" />
-		<!--- TODO : <meta name="description" content="" />-->
+		<meta name="description" lang="sk" content="Skontroluj svoju prácu, či sa nezhoduje s prácami iných.">
+		<meta name="description" lang="en" content="Check your files before submitting.">
 
 		<!-- Facebook Meta Tags -->
-		<!-- TODO: <meta property="og:url" content="" />-->
+		<meta property="og:title" content="Antiplag">
+		<meta property="og:description" lang="sk" content="Skontroluj svoju prácu, či sa nezhoduje s prácami iných.">
+		<meta property="og:description" lang="en" content="Check your files before submitting.">
+		<meta property="og:url" content="process.env.VUE_APP_API_URL">	
+		
 		<meta property="og:type" content="website" />
-		<!-- TODO: <meta property="og:title" content="" />-->
-		<!-- TODO: <meta property="og:description" content=""/>-->
 		<meta
 			property="og:image"
 			content="<%= htmlWebpackPlugin.files.favicon %>"
 		/>
 
 		<!-- Twitter Meta Tags -->
-		<!-- TODO: <meta property="twitter:url" content="" />-->
-		<meta name="twitter:card" content="summary_large_image" />
-		<!-- TODO: <meta name="twitter:title" content="" />-->
-		<!-- TODO: <meta content="" name="twitter:description" />-->
+		<meta name="twitter:title" content="Antiplag">
+		<meta name="twitter:description" lang="sk" content="Skontroluj svoju prácu, či sa nezhoduje s prácami iných.">
+		<meta name="twitter:description" lang="en" content="Check your files before submitting.">
+		<meta name="twitter:card" content="antiplag_large_image" />
+		<meta property="twitter:url" content="process.env.VUE_APP_API_URL" />
```

Comment:
Does this work? I don't see any interpolation here, if you run it in your browser, will it fill the correct value, or will it just paste `"process.env.VUE_APP_API_URL"` ?

![image](https://user-images.githubusercontent.com/27869979/114667051-4ac5c280-9cff-11eb-9d7f-e4478c1a9c65.png)